### ISSUE TITLE
Add zmughal as a core dev

### ISSUE THREAD

**Issue by jberger** [Perl5-Alien/Perl5-Alien-Documents#2]
We have another willing volunteer. @zmughal (sivoais on irc (#pdl)). As we currently have three members on the team, one vote (in addition to mine) is needed to add him.

Note that I would also support adding most of those people would who have submitted pull requests, especially  @plicease, who I have found out has stopped using AB due to lack of certain features (which I have not had time to investigate).

**Comment by jberger** [comment]
Hmmmmmm, just thought about it some more. Should this be a secret vote? Probably? Contact me with your vote and I will announce the result.

**Comment by zmughal** [comment]
Thank you for the nomination, @jberger.

I have been using Alien::Base for a while and have released a few modules to [CPAN](https://metacpan.org/author/ZMUGHAL) that are based on Alien::Base. My interest is making sure that Alien modules can install on as many platforms as possible. I have already contributed to Alien::Base [once in the past](https://github.com/Perl5-Alien/Alien-Base/commits?author=zmughal).

**Comment by plicease** [comment]
(not directly related to @zmughal 's nomination, just commenting since my ears are burning)

I've started working on alternatives to AB because I needed to work around enough problems that it made sense for me (experimentation on different methods is good anyway).  The fact that PR 30 still has not been merged has been a stumbling block for me, since from my perspective it fixes an obvious bug.  I just haven't been submitting patches for other problems that seem really obvious to me since there seems very little hope that they will ever be accepted.

I'd like to see AB as viable though, and am interested collaboration or at least coordination.  Here are some things that I'd be interested in, and may even have time to work on if there is consensus on a path forward:
- static libraries should be used for XS instead of shared libraries to make XS modules less fragile when upgrades happen (see issue 11), I can explain in more detail why I think this is necessary if anyone is interested
- dynamic libraries should be available for FFI modules (an interface for finding the .so and .dll files would be great)
- choice for the downstream Foo::XS module (e.g. use Alien::Foo to get libfoo > 1.2, even if libfoo 1.0 is provided by OS)
- support for windows (I raised one issue with PR 32, which may not be the right solution for that exact problem, but even once that is resolved, there are many many many issues in this can of worms)
- same for cygwin
- less dependence on pkg-config since it is not standard in many environments and not used by all libraries
- use PkgConfig (pure perl replacement for pkg-config now being actively maintained by myself) when pkg-config is not available

I have addressed all of these to varying degrees of success with Alien::Libarchive (and others) and would love to have ideas stolen where appropriate if it improves the state of Alien:: in Perl.
